charts/kubewarden-crds/templates/policyserver.yaml

```diff
@@ -6,6 +6,16 @@ metadata:
     controller-gen.kubebuilder.io/version: v0.4.1
   name: policyservers.policies.kubewarden.io
 spec:
+  conversion:
+    strategy: Webhook
+    webhook:
+      clientConfig:
+        service:
+          name: webhook-service
+          namespace: system
+          path: /convert
```

Comment:
We don't do version conversion, this should not be added I think.

Reply:
Indeed you are right!
Thanks for noticing. Opened https://github.com/kubewarden/kubewarden-controller/pull/196 so this is fixed on next controller releases, and removed and force-pushed in this pr.